### PR TITLE
base-control english version

### PR DESCRIPTION
**Pull request by innabelaya**

### CODE REVIEW

File: common.blocks/base-control/base-control.en.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# base-control
+
+Auxiliary block used for most blocks work of [bem-components](https://github.com/bem/bem-components) library: [attach](https://github.com/bem/bem-components/blob/v2/common.blocks/attach/attach.en.md), [button](https://github.com/bem/bem-components/blob/v2/common.blocks/button/button.en.md), [checkbox](https://github.com/bem/bem-components/blob/v2/common.blocks/checkbox/checkbox.en.title.txt), [input], [link], [menu], [radio].
+
+Designed to implement the common functionality of these blocks, providing `focused` and `disabled` states.
+
+The **base-control** block is designed to implemnt the common functionality of these blocks to provide their `focused` and `disabled` state.
```
Comment (persidskiy) [inline common.blocks/base-control/base-control.en.md]: implement

This paragraph duplicates the previous one, isn't it?

File: common.blocks/base-control/base-control.en.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    </tr>
+    <tr>
+        <td>`getName`</td>
+        <td>Returns the name of the control (if available). If a control name is not available, returns an empty string.</td>
+    </tr>
+    <tr>
+        <td>`getVal`</td>
+        <td>Returns control value.</td>
+    </tr>
+</table>
```
Comment (persidskiy) [inline common.blocks/base-control/base-control.en.md:28]: Is it really convenient using html-tables instead [gfm-style-tables](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#tables)?

File: common.blocks/base-control/base-control.en.md (hunk 1 above)
Comment (innabelaya) [inline common.blocks/base-control/base-control.en.md]: You are right! Thank you! Corrected

File: common.blocks/base-control/base-control.en.md (hunk 2 above)
Comment (innabelaya) [inline common.blocks/base-control/base-control.en.md:28]: Yes, this question was discussed locally. It is not convenient to use gfm-style-tables. Every time you change something you have to correct it to make its view readable. With code samples it is not convenient.

**Pull request merged**